homeassistant/components/bryant_evolution/climate.py

```diff
@@ -0,0 +1,244 @@
+"""Support for Bryant Evolution HVAC systems."""
+
+from datetime import timedelta
+import logging
+from typing import Any
+
+from evolutionhttp import BryantEvolutionClient
+
+from homeassistant.components.climate import (
+    ClimateEntity,
+    ClimateEntityFeature,
+    HVACAction,
+    HVACMode,
+)
+from homeassistant.const import UnitOfTemperature
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import BryantEvolutionConfigEntry
+from .const import CONF_SYSTEM_ID, CONF_ZONE_ID, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+SCAN_INTERVAL = timedelta(seconds=60)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: BryantEvolutionConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up a config entry."""
+    client = config_entry.runtime_data
+    climate = BryantEvolutionClimate(
+        # Suffix with -climate since we expect to add more entities later.
```

Comment:
Expect more climate entities?

Reply:
no, a few sensors.

Reply:
(like, in a future PR)

Reply:
You don't have to prepend it with climate then. They should be unique per integration per platform